ci_example_python/doc/python_code_guidelines.md

```diff
@@ -9,24 +9,14 @@ guidelines are used in the
 [Open Dynamic Robot Initiative](https://open-dynamic-robot-initiative.github.io/)
 
 The following rules present basic guidelines for our **Python** code.
-The goal is to have code that is formatted in a consistent and easily readable
-way while at the same time not being overly complicated by specifying every
-detail. For such guidelines to be practical, newcomers should be able to read
-them within a few minutes and be able to memorize them. So these rules
-intentionally do not cover every detail but rather aim at specifying only the
-big, important things.
-
-If this is too simple for you and you want more rules, you are encouraged to
-read the [PEP 8](https://www.python.org/dev/peps/pep-0008/) on which these rules
-are based.
 
 These guidelines may evolve in time so it is first
 good practice to check them upon creation of a new package or code refactoring.
 
 ## II. Folder Structure and File Naming
 
 - Only one **Python** module per git repository, and it must be located in
```

Comment:
As already mentioned above, I think it should say "Python package" here instead of module.